### PR TITLE
Initial docs required to run the 2022 SC election

### PR DESCRIPTION
**Pull request by geekygirldawn**
/hold

Here are the initial docs for the 2022 Knative SC election.

This PR should not be merged until:

* @knative/steering-committee has approved the timeline and confirmed the number of seats up for election along with the other details.
* Steering will also need to approve a 3rd Election Officer so that I can add them to these docs.
* @jberkus will need to confirm that all of this looks good and that I haven't mangled anything that Elekto uses.

### CODE REVIEW

File: elections/2022-SC/election.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Knative 2022 Steering Committee Election
+organization: Knative
+start_datetime: 2022-11-10 00:00:01
+end_datetime: 2022-11-30 23:59:59
+no_winners: 2
```
Comment (jberkus) [inline elections/2022-SC/election.yaml:5]:
```suggestion
no_winners: 3
```

**Comment by jberkus** [comment]
I thought three seats were open?  If so, I have a bunch of edits.

**Comment by vaikas** [comment]
Only 2 seats (Paul and I). @thisisnotapril kindly stayed in the SC to help with the transition to CNCF, so that seat will not be renewed. We're going to go back to 5 members in the Steering.

**Review by vaikas — COMMENTED** [review]
Thank you so much for doing this!
Couple of nits.

File: elections/2022-SC/election.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+allow_no_opinion: True
+delete_after: True
+show_candidate_fields:
+  - affiliation
+election_officers:
+  - geekygirldawn
+  - jberkus
+eligibility: All Knative contributors with more than 50 contributions are eligible to vote
+exception_description: If you should be eligible to vote, and find that you are unable to, please fill out the exceptions form.    Otherwise, contact elections@knative.team.
```
[the remark this answers is not in the document]
Comment (geekygirldawn) [reply elections/2022-SC/election.yaml]: Fixed - thank you!

File: elections/2022-SC/README.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+| December 6   | Announcement of Results |
+
+## Election Officers
+
+In an effort to run a fair and transparent election, the following people
+have been asked by the Steering Committee to run this election:
+
+- [Josh Berkus](https://github.com/jberkus)
+- [Dawn Foster](https://github.com/geekygirldawn)
+- TBD
```
Comment (csantanapr) [inline elections/2022-SC/README.md:50]: @upodroid and @nader-ziada showed interest to help as officers in Slack

Comment (geekygirldawn) [reply elections/2022-SC/README.md:50]: Exactly! But, the Steering Committee needs to approve / appoint new Election Officers, and we only need one more election officer, so I'm waiting on the official decision from the SC about which one they have selected :)

Comment (lance) [reply elections/2022-SC/README.md:50]: @geekygirldawn looking at the [STEERING-COMMITTEE.md](https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md#election-officers) doc, it's not clear to me if there needs to be a PR of some sort to appoint the election officer. However, in the Steering Committee Slack channel @upodroid was agreed on by 3 members (myself, @csantanapr and @vaikas). Not sure if that is considered quorum given that there are 6 members at the moment. /cc @pmorie @thisisnotapril @itsmurugappan

Comment (jberkus) [reply elections/2022-SC/README.md:50]: Yeah, you need one more vote, please.

**Comment by jberkus** [comment]
Elekto parts look correct.

File: elections/2022-SC/README.md (hunk 3 above)
Comment (csantanapr) [reply elections/2022-SC/README.md:50]: Steering appoints @upodroid as election officer, we just had quorum 
@geekygirldawn and @jberkus please move forward and onboard @upodroid 
Thank you @upodroid for volunteering 👍

**Comment by geekygirldawn** [comment]
I've added our 3rd election officer, so I think this is ready to go now if someone could merge it when you get a chance.

**Comment by geekygirldawn** [comment]
@vaikas - we should definitely talk about the EU seat. We clarified it a bit in the [SC doc](https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md#candidate-eligibility) to say that "The End User seat is appointed by Steering Committee", and last year that took effect in February. I think part of the confusion last year was that we talked about it in the election materials despite it not being included in the election. I'd prefer to leave it out of these docs for now, and once the new SC is seated, the SC can announce the process for appointing someone to the EU seat. 

I think the other part of the confusion last year was that the SC treated it like an election, but with the SC as the people voting. It might be better to use a different process this year that makes it clear that this person will be appointed, and not elected.

After looking at this again, I think I'm going to propose separating the EU info out of the "Election" section of the doc and into a new section to help avoid both of these confusions :)

**Review by pmorie — APPROVED** [review]

**Pull request merged**